### PR TITLE
Feat [EGB-022] create specialities

### PR DESCRIPTION
**Pull request by stanislav-plisko**

### CODE REVIEW

**Comment by Sergick12** [comment]
change nama PR, dont use russian
template: Feat [EGB-002] add gems, devise, rolify, cancancan

File: .rubocop.yml [hunk 1]
```diff
@@ -13,6 +13,8 @@ AllCops:
     - "config/**/*"
     - "Gemfile"
 
+Rails/UniqueValidationWithoutIndex:
```
Comment (Sergick12) [inline .rubocop.yml:16]: why did you change this in this branch? 
rollback  this!

File: Gemfile [hunk 2]
```diff
@@ -56,8 +56,6 @@ group :development, :test do
 
   gem 'byebug'
 
-  gem 'lefthook'
```
Comment (Sergick12) [inline Gemfile:59]: it too

File: Gemfile.lock [hunk 3]
```diff
@@ -142,7 +142,6 @@ GEM
       activesupport (>= 5.0.0)
     json (2.7.2)
     language_server-protocol (3.17.0.3)
-    lefthook (1.6.10)
```
Comment (Sergick12) [inline Gemfile.lock:145]: it too

File: Gemfile.lock [hunk 4]
```diff
@@ -334,7 +333,6 @@ DEPENDENCIES
   faker!
   importmap-rails
   jbuilder
-  lefthook
```
Comment (Sergick12) [inline Gemfile.lock:337]: it too

File: spec/models/speciality_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Speciality do
```
Comment (Sergick12) [inline spec/models/speciality_spec.rb:3]: where are tests?

File: spec/models/student_spec.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Student do
```
Comment (Sergick12) [inline spec/models/student_spec.rb:3]: it too

**Review by Sergick12 — CHANGES_REQUESTED** [review]

**Comment by stanislav-plisko** [comment]
not valid

**Pull request closed without merging**